### ISSUE TITLE
Python3 installation error

### ISSUE THREAD

**Issue by shkarupa-alex** [bakwc/JamSpell#2]
```
 File "/private/var/folders/vz/ybx331j17wxd9z6994w96rkc0000gn/T/pip-build-vbwbjsut/jamspell/setup.py", line 46, in find_swig
        assert subprocess.check_output([swigBinary, "-version"]).find('SWIG Version 3') != -1
    TypeError: a bytes-like object is required, not 'str'
```

**Comment by bakwc** [comment]
thanks for update, i will upload fixed version tonight

**Comment by bakwc** [comment]
Try now, uploaded a fixed version (0.0.9).

**Comment by yoptar** [comment]
It works now, thanks!
